packages/angular/test/README.md

```diff
@@ -108,9 +108,9 @@ Note: You may encounter some other peer dependency issues not covered by the Ang
 `src/app/form/form.component.ts`
 `src/app/modal-example/modal-example.component.ts`
 
-5. Create a directory in `apps` named `ng14`.
-6. Copy the modified files to the `apps/ng14` directory.
+5. Create a directory in `apps` named `ng17`.
+6. Copy the modified files to the `apps/ng17` directory.
 7. Open `./github/workflows/build.yml` and find the `test-angular-e2e` job.
 8. Find the `apps` field under `matrix`.
-9. Add "ng14" to the `apps` field.
+9. Add "ng17" to the `apps` field.
 10. Commit these changes and push.
```

Comment:
I think these should probably stay the same, since they're continuing the example of making a new test app for Angular 14. (Either that, or the sections above that reference 13 & 14 should change, but I think it might be easiest to just keep them with these old numbers, since they're referring to making a new app anyway.)

Reply:
Thank you for catching that! I did a find and replace not realizing that section was showing how to add a new one, ha. Fixed here: https://github.com/ionic-team/ionic-framework/pull/28943/commits/fe3da871bf2fd1b08ad20b8c39fa5369768f7941